stages/03_process_data.py

```diff
@@ -0,0 +1,71 @@
+# Imports
+# -------
+import re
+import pathlib
+import pandas as pd
+
+extensions = ['zip']
+extensions_re = re.compile(r'\.(' + '|'.join(re.escape(ext) for ext in extensions) + r')$')
+files = filter( lambda item: item.is_file(), pathlib.Path('download').rglob('*'))
+
+brick_dir = pathlib.Path('brick')
+brick_dir.mkdir(exist_ok=True)
+
+smiles_list = []
+inhibitor_constant_values = []
+inhibitory_concentration_values = []
+half_maximal_effective_concentrations = []
+dissociation_constants = []
+protein_targets = []
+active_or_inactive = []
+
+for file in files:
+    out_basename = re.sub(extensions_re, '.parquet', file.name )
+    out_file = brick_dir / file.relative_to('download').with_name( out_basename )
+
+    if file.match('*.ism'):
```

Comment:
For the `.ism` files, try the following:

```python
import pandas as pd
df = pd.read_csv('download/aa2ar_active.ism/actives_nM_combined.ism', sep='\0', header=None)
df[0].str.split(' ', n=9, expand=True)
# TODO assign headers for each of the 10 columns (e.g., SMILES, Endpoint, Relation, Value, Unit, etc.)
```

I don't think further processing on the files should be done here (such as grouping). The contents of the Parquet should be as close to the source data as possible.